Activities/Cryptography/UiPath.Cryptography/CryptographyHelper.cs

```diff
@@ -260,5 +298,124 @@ private static int[] GetLegalKeySizes(SymmetricAlgorithm algorithm)
 
             return keySizes.ToArray();
         }
+
+        private static byte[] EncryptAesGcm(byte[] inputBytes, byte[] key, out byte[] result)
```

Comment:
Any good reason why we have result as an out argument and also as a return value?
Can we have it just as a return value?

Same for decrypt method.

Reply:
removed as out parameter